src/app/content/practiceQuestions/components/Answer.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+import { h4Style } from '../../../components/Typography';
+import theme from '../../../theme';
+import ContentExcerpt from '../../components/ContentExcerpt';
+import { practiceQuestionStyles } from '../../constants';
+import { LinkedArchiveTreeSection } from '../../types';
+import { PracticeAnswer, PracticeQuestionStyles } from '../types';
+
+const correctAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'correct');
+const incorrectAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'incorrect');
+const selectedAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'selected');
+const unselectedAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'unselected');
+
+interface AnswerProps {
+  isSelected?: boolean;
+  isSubmitted: boolean;
+  showCorrect?: boolean;
+  answer: PracticeAnswer;
+  index: number;
+  onSelect?: () => void;
+  source: LinkedArchiveTreeSection;
+}
+
+const getChoiceLetter = (value: number) => {
+  return (value + 10).toString(36);
+};
+
+// tslint:disable-next-line: variable-name
+const AnswerExcerpt = styled(ContentExcerpt)`
```

Comment:
i'm on the fence about whether this needs to be a `ContentExcerpt`, it does have math sometimes, but i don't think it has any other type of content or styling